smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/ReactiveKafkaConsumer.java

```diff
@@ -229,6 +229,34 @@ public ConsumerRebalanceListener getRebalanceListener() {
         }
     }
 
+    @Override
+    @CheckReturnValue
+    public Uni<ConsumerGroupMetadata> consumerGroupMetadata() {
+        return runOnPollingThread((Function<Consumer<K, V>, ConsumerGroupMetadata>) Consumer::groupMetadata);
+    }
+
+    @Override
+    @CheckReturnValue
+    public Uni<Void> resetToLastCommittedPositions() {
+        return runOnPollingThread(c -> {
+            Set<TopicPartition> assignments = c.assignment();
+            c.pause(assignments);
+            Map<TopicPartition, OffsetAndMetadata> committed = c.committed(assignments);
+            for (TopicPartition tp : assignments) {
+                OffsetAndMetadata offsetAndMetadata = committed.get(tp);
+                if (offsetAndMetadata != null) {
+                    // Seek to next offset position
+                    c.seek(tp, offsetAndMetadata.offset());
+                } else {
+                    // Seek to beginning of the topic partition
+                    c.seekToBeginning(Collections.singleton(tp));
```

Comment:
I am wondering if we should enforce the reset strategy in this case. I'm happy with the current choice, but maybe mention that this is done on purpose.

Reply:
@cescoffier what do you mean by "enforce the reset strategy"? 
With @Ladicek we discussed this before [here](https://github.com/smallrye/smallrye-reactive-messaging/pull/1680#discussion_r839659815), but I am not convinced of the usefulness of the alternative I proposed. 
For now, I'll add more documentation for this.